download-basket/src/main/java/downloadbasket/data/LoadZipDetails.java

```diff
@@ -0,0 +1,61 @@
+package downloadbasket.data;
+
+public class LoadZipDetails {
```

Comment:
From all the setters in this class only setWFSUrl() and setTemporaryDirectory() actually do what you probably think they do. All the others just assign the variables value to it's own value meaning the setters don't do anything at all.

Reply:
Here's a test for you:
```
package downloadbasket.data;

import org.junit.Test;

import static org.junit.Assert.*;

public class LoadZipDetailsTest {

    final String TESTING = "hello world";

    @Test
    public void testSetWFSUrl() throws Exception {
        LoadZipDetails details = new LoadZipDetails();
        details.setWFSUrl(TESTING);
        assertEquals(details.getWFSUrl(), TESTING);
    }

    @Test
    public void testSetUserEmail() throws Exception {
        LoadZipDetails details = new LoadZipDetails();
        details.setUserEmail(TESTING);
        assertEquals(details.getUserEmail(), TESTING);

    }
}
```